baseline/client/js/jspsych-n-back.js

```diff
@@ -0,0 +1,113 @@
+jsPsych.plugins["n-back"] = (() => {
+    const plugin = {};
+
+    plugin.info = {
+        name: "n-back",
+        parameters: {
+            n: {
+                type: jsPsych.plugins.parameterType.INT,
+                default: undefined,
+            },
+            sequence: {
+                type: jsPsych.plugins.parameterType.STRING,
+                array: true,
+                default: undefined,
+            },
+            show_duration: {
+                type: jsPsych.plugins.parameterType.INT,
+                default: undefined,
+            },
+            hide_duration: {
+                type: jsPsych.plugins.parameterType.INT,
+                default: undefined,
+            },
+            finish_duration: {
+                type: jsPsych.plugins.parameterType.INT,
+                default: 0,
+            },
+        },
+    };
+
+    plugin.trial = (display_element, trial) => {
+        // build and show display HTML
+        {
```

Comment:
I think I must be missing something, because I'm not clear on what this curly brace is doing here. Can you explain?